lighthouse-core/gather/gatherers/start-url.js

```diff
@@ -29,33 +30,27 @@ class StartUrl extends Gatherer {
 
         return this._attemptManifestFetch(passContext.driver, startUrlInfo.startUrl);
       }).catch(() => {
-        return {statusCode: -1, explanation: 'Unable to fetch start URL via service worker'};
+        return {statusCode: -1, explanation: 'Unable to fetch start URL via service worker.'};
```

Comment:
periods added to look better in the report :) (this is currently the only audit using this gatherer)